app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None | str:
+    if not command.startswith("cp ") or len(command.split(" ")) != 3:
+        return "Entered incorrect command!!!"
+    old_file_name = command.split(" ")[1]
+    new_file_name = command.split(" ")[2]
```

Comment:
make ur code DRY, u are using split for 3 times 
also no need to use space inside split method, it uses space by default
also u could unpack ur var like this `var1, var2, var3 = command.split()`